src/renderer/views/pools/Pools.types.ts

```diff
@@ -17,11 +17,14 @@ export type PricePoolAssets = PricePoolAsset[]
 
 export type PricePoolCurrencyWeights = Record<string, number>
 
+// TODO (@asgdx-team) Move all PricePool* types into `src/renderer/services/midgard/types.ts`
 export type PricePool = {
   asset: PricePoolAsset
   poolData: PoolData
 }
 
+export type PricePool$ = Rx.Observable<PricePool>
```

Comment:
just can not see any mentions of this type. do we need this ?

Reply:
Good find! _It's a stale type defination - I will remove it ..._